modules/godot/nodes/script_ecs.cpp

```diff
@@ -34,6 +36,7 @@ ScriptEcs::~ScriptEcs() {
 Vector<StringName> ScriptEcs::spawner_get_components(const StringName &spawner_name) {
```

Comment:
I think that would make more sense wrap the entire function with `ifdef` since this function is supposed to be executed only in editor. Otherwise it's confusing, since it seems that changes behavior when execution in production.